adapter/operations.go

```diff
@@ -14,19 +14,42 @@
 
 package adapter
 
-import "context"
+import (
+	"context"
+	"net/url"
+
+	"github.com/layer5io/meshkit/utils"
+)
 
 var (
-	NoneVersion = []Version{"none"}
+	NoneVersion  = []Version{"none"}
```

Comment:
Oh yes, this is just for a default value. Ideally any implementation should override this. Just a safety check in case if it doesn't override, to avoid panics.

Reply:
Ah ok, a little comment might help?

Reply:
@kumarabd it cycled back to me... code should be designed to handle empty arrays, without panicking. an empty array already means "none". if one needs / want to check, just check it's length. arrays are most often looped through, thus for an empty array
```
for element in array:
   apply(element)
```
will as intended just do nothing. But if it contains "none", it will crash. If it isn't overridden, the result should be that nothing happens. Also, one doesn't need to remember to remove "none" if one just want to append an element. 
So I think the default value should be an empty array :smile: